src/components/MultiSelect/multiselect.scss

```diff
@@ -0,0 +1,8 @@
+@import "~@blueprintjs/select/src/components/multi-select/multi-select";
+
+.#{$ns}-multi-select-popover {
+    .#{$ns}-menu {
+        max-width: unset;
+        max-height: var(--multisuggest-max-height, 45vh);
```

Comment:
Please use a naming schema that reflects the origin of the custom property, e.g. `--eccgui-multisuggestfield-max-height`.